pkg/gateway/concurrent/const.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019-2024 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package concurrent
+
+// UnixSocketPath is the path of the Unix socket.
+const unixSocketPath string = "/ipc/leader.sock"
+
+// ContainerNames is the list of container names.
+var containerNames = []string{"wireguard", "geneve"}
```

Comment:
Should we make these configurable and not hardcoded? So that we better support different technologies and to ease eventual future changes